packages/mobile/src/screens/audio-screen/AudioScreen.tsx

```diff
@@ -0,0 +1,103 @@
+import { Dimensions, View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+import { Text, Tile } from 'app/components/core'
+import { Header } from 'app/components/header'
+import { makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+const messages = {
+  title: 'AUDIO & Rewards'
+}
+
+const screenHeight = Dimensions.get('window').height
+
+const useStyles = makeStyles(() => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    height: screenHeight
+  },
+  tiles: {},
+  tileRoot: {
+    margin: 12
```

Comment:
can use spacing here

Reply:
it is bothersome to me that borderRadius & spacing are not related now. we must fix!!!